### PR TITLE
[EA] Spotlight enhancements April '24

### PR DESCRIPTION
**Pull request by oetherington**
[Figma](https://www.figma.com/file/qAHRGADVOGojqKpqbGMFFu/2024-Q2?node-id=450%3A22707&mode=dev)

Screenshot 2024-04-24 at 19 03 27

Screenshot 2024-04-24 at 19 03 19

Screenshot 2024-04-24 at 19 02 03

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1207162035569970) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/spotlights/SpotlightItem.tsx [hunk 1]
```diff
@@ -328,7 +366,7 @@ export const SpotlightItem = ({
         {hideBanner && <div className={classes.closeButtonWrapper}>
           <LWTooltip title="Hide this spotlight" placement="right">
             <Button className={classes.closeButton} onClick={hideBanner}>
-              <CloseIcon className={classes.closeIcon} />
+              <ForumIcon icon="Close" />
```
Comment (oetherington) [inline packages/lesswrong/components/spotlights/SpotlightItem.tsx:369]: The `closeIcon` class doesn't exist

File: packages/lesswrong/components/spotlights/SpotlightItem.tsx [hunk 2]
```diff
@@ -272,14 +294,30 @@ export const SpotlightItem = ({
...
     setEdit(false);
     refetchAllSpotlights?.();
-  };
-  
+  }, [refetchAllSpotlights]);
+
+  // Define fade color with a CSS variable to be accessed in the styles
+  const style = {
+    "--spotlight-fade": spotlight.imageFadeColor,
+  } as CSSProperties;
```
Comment (oetherington) [inline packages/lesswrong/components/spotlights/SpotlightItem.tsx:305]: This is the same technique we used for `ForumEvents`

**Comment by agnesstenlund** [comment]
Looks great!! 
Screenshot 2024-04-24 at 18 22 02

- [ ]  could we reuse the smaller `x` from the big event banner? (not blocking merging)

Screenshot 2024-04-24 at 18 22 11

**Comment by agnesstenlund** [comment]
- [ ] also think the fade needs to be sliiightly wider, more like half faded than it looks now

**Comment by agnesstenlund** [comment]
- [ ] oh also the link should probably shouldn't change on hover, or only go from white to light grey
Screenshot 2024-04-24 at 18 27 20

**Comment by oetherington** [comment]
@agnesstenlund Done!

Screenshot 2024-04-24 at 20 23 23

File: packages/lesswrong/components/spotlights/SpotlightItem.tsx [hunk 3]
```diff
@@ -115,37 +134,59 @@ const styles = (theme: ThemeType): JssStyles => ({
...
+      maxWidth: TEXT_WIDTH,
     } : {
+      color: theme.palette.grey[700],
       fontSize: 15,
       marginTop: -1,
     }),
   },
   startOrContinue: {
-    marginTop: isFriendlyUI ? 16 : 4,
+    marginTop: isFriendlyUI ? 0 : 4,
```
Comment (s-cheng) [inline packages/lesswrong/components/spotlights/SpotlightItem.tsx:189]: Might need some margin here for the mobile version:

Screen Shot 2024-04-24 at 2 16 12 PM

**Review by s-cheng — APPROVED** [review]
LGTM 👍

**Comment by agnesstenlund** [comment]
Looks perfect! 👌

**Pull request merged**